include/exadg/functions_and_boundary_conditions/interface_coupling.cpp

```diff
@@ -60,14 +61,45 @@ InterfaceCoupling<rank, dim, Number>::setup(
                               return marked_vertices_src_;
                             }));
 
-    map_evaluator[quad_index].reinit(interface_data_dst->get_array_q_points(quad_index),
-                                     dof_handler_src_.get_triangulation(),
-                                     mapping_src_);
-
-    AssertThrow(
-      map_evaluator[quad_index].all_points_found() == true,
-      dealii::ExcMessage(
-        "Setup of InterfaceCoupling was not successful. Not all points have been found."));
+    auto const * points = &interface_data_dst->get_array_q_points(quad_index);
+
+    map_evaluator[quad_index].reinit(*points, dof_handler_src_.get_triangulation(), mapping_src_);
+
+    if(not map_evaluator[quad_index].all_points_found())
+    {
+      // get vector of points not found
+      std::vector<dealii::Point<dim>> points_not_found;
+      points_not_found.reserve(points->size());
+      unsigned int n_points_not_found = 0;
+      for(unsigned int i = 0; i < points->size(); ++i)
+      {
+        if(not map_evaluator[quad_index].point_found(i))
+        {
+          n_points_not_found += 1;
+          points_not_found.push_back((*points)[i]);
+        }
+      }
+      n_points_not_found =
+        dealii::Utilities::MPI::sum(n_points_not_found, dof_handler_src->get_communicator());
```

Comment:
`points_not_found()` and `n_points_not_found()` could be two useful additional member functions of `dealii::RPE` for the future.

Reply:
As a reminder, I opened an issue in dealii: https://github.com/dealii/dealii/issues/15248